vrc-get-gui/src/config.rs

```diff
@@ -33,8 +33,6 @@ pub struct GuiConfig {
     #[serde(default = "log_level_default")]
     pub logs_level: Vec<LogLevel>,
     #[serde(default)]
```

Comment:
I meant true by default for serde but it's ok to disable by default.

(I feel if it's disabled by default, no one will use the option so on by default would be better, or always enabled can be a option since animation is fast enough I feel)